node/src/action.rs

```diff
@@ -57,6 +57,7 @@ pub enum Action {
     RpcEffectful(RpcEffectfulAction),
 
     WatchedAccounts(WatchedAccountsAction),
+    Exit(ExitAction),
```

Comment:
There is no documentation for the others. Could you add a one-liner please?

Also, why do you call it `Exit`? By reading the code, I would have expected that it is an action to kill the node. But it is simply writing an error message. It is inconsistent with what we would expect by reading the code.